src/SoundManager.h

```diff
@@ -34,6 +34,9 @@ OSErr SndNewChannel(SndChannelPtr* chan, uint16_t synth, int32_t init, void* use
 OSErr SndDoImmediate(SndChannelPtr chan, const SndCommand* cmd);
 OSErr SndPlay(SndChannelPtr chan, Handle sndHdl, Boolean async);
 
+OSErr SetDefaultOutputVolume(uint32_t level);
+#define SetSoundVol SetDefaultOutputVolume
```

Comment:
👍🏻 Nice, was about to write a comment that Realmz's code didn't use `SetDefaultOutputVolume` directly, but as usual, you are two steps ahead of me. 😄